services/notifications/Dockerfile

```diff
@@ -28,6 +26,7 @@ RUN --mount=type=cache,target=/var/cache/apt,sharing=private \
   apt-get install -y --no-install-recommends \
   gosu \
   && apt-get clean -y \
+  && rm -rf /var/lib/apt/lists/* \
```

Comment:
please double check, doing this I think destroy the use of line 23, which is to keep the cache of apt downloaded files.